scripts/tests/run_test_suite.py

```diff
@@ -123,8 +123,8 @@ class RunContext:
 )
 @click.option(
     '--runner',
-    type=click.Choice(['codegen', 'chip_repl_python', 'chip_tool_python'], case_sensitive=False),
-    default='codegen',
+    type=click.Choice(['chip_repl_python', 'chip_tool_python', 'darwin_framework_tool_python'], case_sensitive=False),
```

Comment:
Followup: should we drop the _python suffixes if they are all python now?  Or at least have aliases where chip_tool is treated as chip_tool_python, etc, if we don't want to break existing scripts but want to reduce the typing?

Reply:
That makes perfect sense... I have opened https://github.com/project-chip/connectedhomeip/issues/31862 as a followup for that.